### PR TITLE
Change the Flutter package to an updated one

### PR DESCRIPTION
**Pull request by jlnrrg**
**Issue:** No issue, just a change in the readme for an updated flutter package

### CODE REVIEW

**Review by ericcornelissen — APPROVED** [review]
This is okay by as the [previous package that was linked](https://github.com/NikSWE/flutter_brand_icons) was last updated 2 years ago and therefore horribly out of date. (And a minor concern is that the authors URL was broken as well.) Still, I'll wait for at least one other @simple-icons/maintainers to look at this before we merge it :slightly_smiling_face:

**Comment by fbernhart** [comment]
LGTM! 😊👍🏼

**Pull request merged**
